### PR TITLE
Applies a code of conduct to tgstation

### PR DESCRIPTION
**Pull request by optimumtact**
We need to reign in the host

### CODE REVIEW

**Review by Cyberboss — DISMISSED** [review]

**Comment by praisenarsie** [comment]
FUCK

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+## Scope
+
+This Code of Conduct applies both within project spaces and in public spaces when an individual is representing the project or its community. Examples of representing a project or community include using an official project e-mail address, posting via an official social media account, or acting as an appointed representative at an online or offline event. Representation of a project may be further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team at abuse@gmail.com. The project team will review and investigate all complaints, and will respond in a way that it deems appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
```
Comment (praisenarsie) [inline CODE_OF_CONDUCT.md]: didn't know tg owned abuse@gmail.com

**Comment by delaron** [comment]
Isn't this on the wiki?

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (Cyberboss) [inline CODE_OF_CONDUCT.md]: #coderbus at irc.rizon.net

**Review by Cyberboss — DISMISSED** [review]

**Comment by JJRcop** [comment]
This changed dramatically over the course of 2 minutes

**Review by MrsTonedOne — APPROVED** [review]

**Review by duncathan — APPROVED** [review]

**Review by ninjanomnom — APPROVED** [review]

**Comment by Buggy123** [comment]
You can't tell me what to do!

**Review by ShizCalev — APPROVED** [review]

**Comment by Arianya** [comment]
>wiki edits

Oh no all those wiki edits we see!!

**Comment by lzimann** [comment]
This latest version is great

**Review by Cyberboss — APPROVED** [review]

**Comment by GunHog** [comment]
This perfectly reflects /tg/, unlike that other silly version! :+1:

**Comment by PKPenguin321** [comment]
pussy maintainers wont delete this comment 😎

**Comment by ShizCalev** [comment]
image
fite me

**Comment by optimumtact** [comment]
deleted it

**Comment by Cyberboss** [comment]
No one will ever know...

**Comment by Cyberboss** [comment]
can't escape meme man

**Review by monster860 — APPROVED** [review]
let's do it

**Comment by ZeWaka** [comment]
please no i like shitposting

**Comment by monster860** [comment]
If you want shitposting, there's two good shitposting clubs available: The yogs coder chat, and the host chat.

**Pull request closed without merging**